### ISSUE TITLE
LevelDB LOCK file is not removed when an app closes - blocks localStorage when an app is reinstalled.

### ISSUE THREAD

**Issue by grantcv1** [electron/electron#18263]
### Preflight Checklist

* [ x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x ] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x ] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

On Windows, a leveldb LOCK file, apparently used for local storage, does not get removed when a packaged and installed electron application closes. The LOCK file appears when the application opens, but does not disappear after app.quit(). This has no adverse effect and the localStorage continues to work between sessions as expected until the application is reinstalled.

Once the application is reinstalled, the LOCK file appears to block access to the leveldb data and localStorage no longer persists. When the stale LOCK file is deleted, access to localStorage is restored and everything works fine -- even old data stored prior to the reinstall becomes available once more.

This problem became apparent to me after I installed electron-builder 20.38.0, which I did as part of my upgrade to electron 5.0.0, so I'm not sure where the real issue lies. I think, but am not sure, that this problem was not present with electron 5.0.0 / electron-builder 20.15.0.

* **Electron Version:**
  * 5.0.0
* **Operating System:**
  * Windows 10
* **Last Known Working Electron version:**:
  * 4.1.5 (possibly 5.0.0 with electron-builder 20.15.0)

### Expected Behavior
localStorage data will persist between application sessions, even when the application is reinstalled.

### Actual Behavior
localStorage access is blocked by a stale LOCK file once an application is reinstalled.

### To Reproduce
I have not been able to reproduce this scenario in a dev environment. However, it's pretty easy to see on a packaged and installed app, that the leveldb LOCK file is not removed whenever the application closes.  If this LOCK file would be removed, presumably by app.quit(), the problem would never exist.

**Comment by electron-triage** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron 6.x.y or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by electron-triage** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.
